vm_supervisor/README.md

```diff
@@ -118,6 +118,10 @@ or in debug mode:
 ```shell
 python3 -m vm_supervisor -vv --system-logs
 ```
+or in test mode to run a single function:
+```shell
+python3 -m vm_supervisor -vv --system-logs -f examples/example_fastapi
+```
```

Comment:
This should be the topic of another pull request. It would not work as is as using the "fake program" mode also requires building a runtime locally as well as an example squashfs volume. That procedure is documented in `/tutorials/TESTING.md`